client/app/components/composites/ListingCard/ListingCard.js

```diff
@@ -56,12 +57,12 @@ class ListingCard extends Component {
         style: { backgroundColor: `rgb(${tintedRGB.r}, ${tintedRGB.g}, ${tintedRGB.b})` },
         href: this.props.listingURL,
       }, this.props.imageURL && this.state.imageStatus !== 'failed' ?
-        img({
+        img(Object.assign({}, {
           className: css.thumbnail,
           src: this.props.imageURL,
           onLoad: this.handleImageLoaded,
           onError: this.handleImageErrored,
-        }) :
+        }, higherRes)) :
```

Comment:
I'm not sure we want to prefer spread on every cases.
Here it would mean something like:
```
{
  ...{
    className: css.thumbnail,
    src: this.props.imageURL,
    onLoad: this.handleImageLoaded,
    onError: this.handleImageErrored,
  },
  ...higherRes,
}
```
Where it is very easy to forget to do that mutation wrapping spread (object assign makes it definitely more clear.)